packages/libs/eda/src/lib/map/analysis/TimeSliderQuickFilter.tsx

```diff
@@ -234,7 +234,11 @@ export default function TimeSliderQuickFilter({
       <p>
         Expand the panel with the{' '}
         <ChevronRight transform={'matrix(0,1,-1,0,0,0)'} /> tab or click{' '}
-        <a style={{ cursor: 'pointer' }} onClick={() => setMinimized(false)}>
+        <a
+          href="/#"
```

Comment:
@bobular Thank you for your reviews 👍 Yes it shows warning like "no href". The "/#" actually do nothing and it is one of recommended ways: otherwise, warning message recommends to use button element, but it is not the case for this.

Reply:
In Firefox the `/#` can show up bottom-left of the screen and I think it can end up in the address bar if clicked on. If you can remove that would be great!

Reply:
Oops, I didn't check Firefox. Okay then I will remove it and merge Thanks @bobular 👍